composer.json

```diff
@@ -3,74 +3,79 @@
     "license": "MIT",
     "type": "project",
     "description": "Symfony Demo Application",
-    "autoload": {
-        "psr-4": {
-            "AppBundle\\": "src/AppBundle/",
-            "CodeExplorerBundle\\": "src/CodeExplorerBundle/"
-        },
-        "classmap": [ "app/AppKernel.php", "app/AppCache.php" ]
-    },
-    "autoload-dev": {
-        "psr-4": { "Tests\\": "tests/" },
-        "files": [ "vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php" ]
-    },
     "require": {
-        "php"                                  : ">=5.5.9",
-        "ext-pdo_sqlite"                       : "*",
-        "doctrine/doctrine-bundle"             : "^1.6",
-        "doctrine/doctrine-cache-bundle"       : "^1.2",
-        "doctrine/doctrine-fixtures-bundle"    : "^2.2",
-        "doctrine/orm"                         : "^2.5",
-        "erusev/parsedown"                     : "^1.5",
-        "ezyang/htmlpurifier"                  : "^4.7",
-        "incenteev/composer-parameter-handler" : "^2.0",
-        "sensio/distribution-bundle"           : "^5.0",
-        "sensio/framework-extra-bundle"        : "^3.0.2",
-        "symfony/monolog-bundle"               : "^3.0",
-        "symfony/polyfill-apcu"                : "^1.0",
-        "symfony/swiftmailer-bundle"           : "^2.3",
-        "symfony/symfony"                      : "^3.3",
-        "twig/extensions"                      : "^1.5",
-        "twig/twig"                            : "^1.28 || ^2.0",
-        "white-october/pagerfanta-bundle"      : "^1.0"
+        "php": "^7.1.3",
+        "ext-pdo_sqlite": "*",
+        "doctrine/doctrine-fixtures-bundle": "^2.3",
+        "erusev/parsedown": "^1.6",
+        "ezyang/htmlpurifier": "^4.9",
+        "sensio/framework-extra-bundle": "^3.0",
+        "sensiolabs/security-checker": "^4.1",
+        "symfony/asset": "^3.3",
+        "symfony/browser-kit": "^3.3",
+        "symfony/css-selector": "^3.3",
+        "symfony/expression-language": "^3.3",
+        "symfony/flex": "^1.0",
+        "symfony/form": "^3.3",
+        "symfony/framework-bundle": "^3.3",
+        "symfony/monolog-bundle": "^3.1",
+        "symfony/orm-pack": "^1.0",
+        "symfony/polyfill-apcu": "^1.4",
+        "symfony/security-bundle": "^3.3",
+        "symfony/swiftmailer-bundle": "^3.0",
+        "symfony/templating": "^3.3",
```

Comment:
you can remove the templating component (revert https://github.com/symfony/symfony-demo/pull/617/commits/e8d0b8a56dd29cde5f5d9335426c38e05c479572), since the error is fixed in 3.3.7.